appcircle_ios_resign/1.0.1/component.yaml

```diff
@@ -0,0 +1,37 @@
+platform: iOS
+buildPlatform:
+displayName: "Resign IPA"
+description: "Resign IPA with different certificates and provisioning profiles."
+docsUrl: 
+requiredComponents: "appcircle_ios_build_sign"
```

Comment:
@boztopuz I am not sure is it required field or not.

Reply:
You cannot re-sign without ipa, for that appcircle_ios_sign must run before the component. The purpose of this is to extract 2 separate ipas signed with 2 different certificates over the same pipeline